docker/validator/docker-run.sh

```diff
@@ -4,9 +4,9 @@
 set -ex
 
 cd /opt/libra/etc
-echo "$NODE_CONFIG" > node.yaml
-echo "$SEED_PEERS" > seed_peers.yaml
-echo "$NETWORK_KEYPAIRS" > network_keypairs.yaml
-echo "$CONSENSUS_KEYPAIR" > consensus_keypair.yaml
-echo "$FULLNODE_KEYPAIRS" > fullnode_keypairs.yaml
+[ -n "${DEPLOY_ENV}" ] && echo "$NODE_CONFIG" > node.yaml
```

Comment:
What is DEPLOY_ENV used for? Can't find reference in the existing codebase

Reply:
Oh, good eye. This is indeed a bug